src/boot/bootctl-status.c

```diff
@@ -373,6 +373,7 @@ int verb_status(int argc, char *argv[], void *userdata) {
                         { EFI_LOADER_FEATURE_SECUREBOOT_ENROLL,       "Enroll SecureBoot keys"                },
                         { EFI_LOADER_FEATURE_RETAIN_SHIM,             "Retain SHIM protocols"                 },
                         { EFI_LOADER_FEATURE_MENU_DISABLE,            "Menu can be disabled"                  },
+                        { EFI_LOADER_FEATURE_MENU_GROUP,              "Menu entries can be grouped"           },
```

Comment:
hmm, so far we only set feature flags of stuf that affects how you can communicate with sd-boot, i.e. whether it checks some efi vars, reads some files from the ESP and so on. But this one is mostly a UI issue, not sure we should stuff this into the feature flags stuff? dunno?